models/eos/ospfv3/eos_ospfv3.yaml

```diff
@@ -0,0 +1,405 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+  {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+  }
+NETWORK_OS: eos
+RESOURCE: ospfv3
+COPYRIGHT: Copyright 2020 Red Hat
+
+DOCUMENTATION: |
+module: eos_ospfv3
+short_description: OSPFv3 resource module
+description: This module configures and manages the attributes of ospfv3 on Arista
```

Comment:
```suggestion
description: This module configures and manages the attributes of OSPFv3 on Arista
```

Reply:
Please make this change througout - ospfv3 should be OSPFv3 except for when it's part of a command/parameter etc.